### PR TITLE
Suggest new code owner policy for Service chain.

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
@smseo @kjhman21 Please consider this new code owner policy.

To develop service chain efficiently and consider the change of other feature which is being developed, 
@KimKyungup @jeonghonah are added to below package.

- /build : related with KSCN/PN/EN package
- /cmd : related with SCN/PN/EN/homi
- /contract : related with bridge/token cotract
- /node : related with main/subbridge

@jeonghonah are added to below package.

- /client : related with main/subbridge

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -29,14 +29,13 @@
 /accounts/         @kjhman21
 /api/              @kjhman21
 /blockchain/       @kjhman21
-/build/            @bitaholic
-/client/           @KimKyungup
-/cmd/              @bitaholic
-/cmd/kscn/         @KimKyungup
+/build/            @bitaholic @KimKyungup @jeonghonah
+/client/           @KimKyungup @jeonghonah
+/cmd/              @bitaholic @KimKyungup @jeonghonah
```
Comment (smseo) [inline .github/CODEOWNERS]: IMO, it would be better to be more specific than assigning many people to a top-level directory.

Comment (jeonghonah) [reply .github/CODEOWNERS]: We agree with the intention. However, the changes in build/node/cmd are necessary to follow-up and if necessary, the change also can be apply to the code of the service chain, so we put it in the top-level reviewer.

Comment (KimKyungup) [reply .github/CODEOWNERS]: During development, to align SCN/SPN/SEN with CN/PN/EN, this way is good. :-)

**Comment by KimKyungup** [comment]
> I still don't see why this PR is needed. If you guys wish to boost the review process, I'd recommend to work on a dev branch, rather than trying to merge PRs directly to the master.

This PR's purpose is not to speed up our development. 
During development, we need to follow up other changes like CN/PN/EN packaging and prevent side effect with service chain code for SCN/SPN/SEN cased by other code.

**Comment by KimKyungup** [comment]
@colin @smseo Please consider this later. :) For now, I close this.

**Comment by smseo** [comment]
@KimKyungup I will let you guys discuss and decide the code owner policy for this repository. I will remove myself from the code owner list and be working as just a contributor. :)

**Pull request closed without merging**